includes/reader-revenue/class-stripe-connection.php

```diff
@@ -422,7 +422,18 @@ private static function get_saved_stripe_data() {
 			$wc_configuration_manager = Configuration_Managers::configuration_manager_class_for_plugin_slug( 'woocommerce' );
 			$stripe_data              = $wc_configuration_manager->stripe_data();
 		} else {
-			$stripe_data = array_merge( $stripe_data, get_option( self::STRIPE_DATA_OPTION_NAME, [] ) );
+			// Filter out empty values, which are not booleans.
```

Comment:
Would this variable ever store an integer `0`? They would be filtered out.

Reply:
No, it's just strings and booleans.